core/state/snapshot/journal.go

```diff
@@ -66,6 +66,19 @@ type journalStorage struct {
 	Vals [][]byte
 }
 
+func ParseGeneratorStatus(generatorBlob []byte) string {
+	if len(generatorBlob) == 0 {
+		return ""
+	}
+	var generator journalGenerator
+	if err := rlp.DecodeBytes(generatorBlob, &generator); err != nil {
+		log.Warn("failed to decode snapshot generator", "err", err)
+		return ""
+	}
+	return fmt.Sprintf(`Wiping: %v, Done: %v, Accounts: %d, Slots: %d, Storage: %d, Marker: %x`,
```

Comment:
Nitpick: 
- we don't have the wiping notion anymore. Wiping will never happen again. I would prefer to not show this info.
- generation marker can indicate to a storage position, which is 2 * 32 bytes length. I would prefer to interpret it a bit further, e.g. 

```go
// Figure out whether we're after or within an account
	switch len(marker) {
	case common.HashLength:
		ctx = append(ctx, []interface{}{"at", common.BytesToHash(marker)}...)
	case 2 * common.HashLength:
		ctx = append(ctx, []interface{}{
			"in", common.BytesToHash(marker[:common.HashLength]),
			"at", common.BytesToHash(marker[common.HashLength:]),
		}...)
	}
```